MotorControl/MotorControl/Resources/Walk_30seconds.json

```diff
@@ -0,0 +1,272 @@
+            "image":{
+                "type":"fetchable",
+                "imageName":"F-TurnUpVolume",
+                "placementType":"topMarginBackground"
+            },
+            "colorTheme":{
+                "colorStyle": {
+                    "header": "darkBackground",
+                    "body": "lightBackground",
+                    "footer": "lightBackground"
+                }
+            },
+            "viewTheme":{
+                "viewIdentifier":"Instruction",
+                "storyboardIdentifier":"ActiveTaskSteps"
+            },
+            "actions":{
+                "goForward":{
+                    "type":"default",
+                    "buttonTitle":"The phone’s volume is up"
+                }
+            }
+        },
+        {
+            "identifier":"pantsWithPocket",
+            "type":"instruction",
+            "title":"Pants with front pockets",
+            "text":"Before continuing on with the short walk, please make sure you have front pockets so you have a place to put the phone. If you do not have pockets, you can place the phone in the waistband of your pants.",
+            "image":{
+                "type":"fetchable",
+                "imageName":"F-PantsPocket",
+                "placementType":"topBackground"
+            },
+            "colorTheme":{
+                "colorStyle": {
+                    "header": "darkBackground",
+                    "body": "lightBackground",
+                    "footer": "lightBackground"
+                }
+            },
+            "viewTheme":{
+                "viewIdentifier":"Instruction",
+                "storyboardIdentifier":"ActiveTaskSteps"
+            },
+            "actions":{
+                "goForward":{
+                    "type":"default",
+                    "buttonTitle":"Got front pockets"
+                }
+            }
+        },
+        {
+            "identifier":"phoneInPocket.1",
+            "type":"instruction",
+            "title":"Put the phone in your pocket",
+            "image":{
+                "type":"animated",
+                "imageNames":["F-PhonePocket1", "F-PhonePocket2", "F-PhonePocket3"],
+                "placementType":"topBackground",
+                "animationDuration":1.5
+            },
+            "colorTheme":{
+                "colorStyle": {
+                    "header": "darkBackground",
+                    "body": "lightBackground",
+                    "footer": "lightBackground"
+                }
+            },
+            "viewTheme":{
+                "viewIdentifier":"Instruction",
+                "storyboardIdentifier":"ActiveTaskSteps"
+            },
+            "text":"This test measures your leg movement and you will need to put your phone in your front pocket. If you do not have pockets you can place the phone in the waistband of your pants.",
```

Comment:
Conditional if statement needs a comma.

"If you do not have pockets, you can place the phone in the waistband of your pants."